### PR TITLE
Blocking wait as function

### PR DESCRIPTION
**Pull request by schleichardt**
@lauraluiz what do you think? @cneijenhuis need this for the payment scenario where 3 requests are performed in parallel but then a blocking wait should take place.

### CODE REVIEW

**Comment by cneijenhuis** [comment]
Nice, thanks :+1:

File: sphere-common/src/main/java/io/sphere/sdk/client/SphereClientUtils.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.util.Optional;
+import java.util.concurrent.*;
+
+public final class SphereClientUtils {
+    private SphereClientUtils() {
+    }
+
+    /**
+     * Waits with a timeout for RESPONSES of a commercetools client wrapped in a {@link CompletionStage}.
+     * This method should not ne used for other {@link CompletionStage}s since it is throwing {@link io.sphere.sdk.models.SphereException}s.
```
Comment (lauraluiz) [inline sphere-common/src/main/java/io/sphere/sdk/client/SphereClientUtils.java]: Small typo: `should not Ne used`

File: src/main/java/io/sphere/sdk/meta/AsyncDocumentation.java [hunk 2]
```diff
@@ -29,6 +29,20 @@
...
+
+ A lot of frameworks do not support asynchronous results but it is still possible to execute calls in parallel and then wait for computed result:
+
+ {@include.example io.sphere.sdk.meta.AsyncDocumentationTest#firstAsyncThenSync()}
+
+ <!--
+
+ thenCompose/thenCombine examples und dann blocking wait
+
+ -->
```
Comment (lauraluiz) [inline src/main/java/io/sphere/sdk/meta/AsyncDocumentation.java]: Should the comment be removed?

**Comment by lauraluiz** [comment]
Looks good. In the end is just moving out the blocking logic to a utils class and keeping the rest the same. I think it's good people use the proper blocking code.

**Pull request merged**